### DISCUSSION TITLE
Best feature to add! (TODO list based)

### DISCUSSION THREAD

**Discussion by odvigajlo** [niels747/2D-Weather-Sandbox#64] [Polls]
title... pretty hard one to choose🤔

**Comment by Meso22** [comment]
I am pretty sure that sim height and precip spawning system are going to be added this weekend and maybe the weather stations and terrain generation might be added to but the rest will be for later on.

**Reply by niels747** [reply]
Sorry, it takes a little longer...

**Comment by Meso22** [comment]
I am dying to get the coriolis effect added so we can simulate very strong upper level wind

**Reply by MET905** [reply]
same. Im going to create bombogenesis lol!
